invenio_records_resources/services/custom_fields/base.py

```diff
@@ -13,12 +13,25 @@
 class BaseCF(ABC):
     """Base Custom Field class."""
 
-    def __init__(self, name):
+    def __init__(self, name, required=False, validate=None, error_messages=None):
```

Comment:
WHat about accepting general `field_kwargs` e.g:
```suggestion
    def __init__(self, name, field_kwargs=None):
```
What's the behaviour when you pass non existing kwargs in marshmallow field?